prover/witness_generator/src/basic_circuits.rs

```diff
@@ -682,24 +681,45 @@ async fn generate_witness(
             VmStorageOracle::new(storage_view.clone());
         let storage_oracle = StorageOracle::new(vm_storage_oracle, storage_refunds);
 
+        const ARRAY_REPEAT_VALUE: std::option::Option<Vec<u8>> = None;
         let (scheduler_witness, block_witness) = zkevm_test_harness::external_calls::run(
             Address::zero(),
             BOOTLOADER_ADDRESS,
             bootloader_code,
             bootloader_contents,
             false,
             account_code_hash,
+            account_code_hash, // 100% wrong, but let's just to unlock compilation?
```

Comment:
This is the `evm_simulator_code`.

it should depend on what we set in BlockMetaParameters (as now it has the third thing after bootloader and aa).

https://github.com/matter-labs/era-zkevm_circuits/blob/v1.5.0/src/scheduler/block_header/mod.rs#L47